client/cli/src/lib.rs

```diff
@@ -295,6 +300,78 @@ impl<'a, CC, RP> ParseAndPrepare<'a, CC, RP> where CC: GetSharedParams {
 	}
 }
 
+impl<'a, CC, RP> ParseAndPrepare<'a, CC, RP> {
+	/// Convert ParseAndPrepare to Configuration
+	pub fn into_configuration<C, G, E, S>(
```

Comment:
Should existing `node/cli` and `node-template/cli` be using that function?

Reply:
(answered on https://github.com/paritytech/substrate/pull/4643#issuecomment-575127599 )